matter_server/server/device_controller.py

```diff
@@ -166,6 +166,7 @@ def __init__(
         self._custom_attribute_poller_timer: asyncio.TimerHandle | None = None
         self._custom_attribute_poller_task: asyncio.Task | None = None
         self._attribute_update_callbacks: dict[int, list[Callable]] = {}
+        self._default_fabric_label: str = "Home Assistant"
```

Comment:
I think a generic name eg the library name is more appropriate. Home Assistant can change this as needed.

Reply:
Yeah, I had "Python Matter Server" first, but then reverted to this.

The reason is that if we release this as a non-breaking change, then folks who have an older Core version will commission devices with that default label.

So we have two options here:
a) Make it a non-breaking change and accept the default fabric label (whichever we are going for)
b) Declare this as a breaking change, bump the minimum Core version in the add-on once we release the first Matter Server with this in.

Maybe a third option is to have no default at all? :thinking: 

c) Declare None as default and not set a label in this case?

Reply:
No label as default sounds ok.

Reply:
Changed accordingly.